bin/install.sh

```diff
@@ -339,6 +346,7 @@ echo -e "\n\nGreat! That's all we need, we will have your Waffle.io Takeout read
 # Store off environment configuration in /etc/waffle #
 ######################################################
 echo -e "\n\nSaving environment configuration in /etc/waffle/"
+sudo mkdir -p /etc/waffle
```

Comment:
`mkdir` throws errors if it already exists, the `-p` makes it not display an error.